.github/workflows/wasm_ts.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
This file does not pass CI. But it's already fixed in https://github.com/nmauersberg/ink-playground/blob/wasm-restructure-skeleton/.github/workflows/wasm_ts.yml

without changing this, we should not be able to merge/rebase the PR.